__init__.py

```diff
@@ -1 +1,3 @@
-from .forgot_password import *
+from .forgot_password import init
+
+init()
```

Comment:
I am not sure about this... have you considered what happen if a developer git submodule import the `forgot_password` repo to their cloudcode directory? this `__init__.py` file will be loaded first